src/main/java/org/jenkinsci/maven/plugins/hpi/AbstractJenkinsManifestMojo.java

```diff
@@ -213,7 +215,11 @@ protected void setAttributes(Manifest.ExistingSection mainSection) throws MojoEx
         addLicenseAttributesForManifest(mainSection);
         addPropertyAttributeIfNotNull(mainSection, "Plugin-ChangelogUrl", "hpi.pluginChangelogUrl");
         addPropertyAttributeIfNotNull(mainSection, "Plugin-LogoUrl", "hpi.pluginLogoUrl");
+
         addAttributeIfNotNull(mainSection, "Plugin-ScmUrl", getScmUrl());
+        addAttributeIfNotNull(mainSection, "Plugin-Scm-Connection", getSCMConnection());
+        addAttributeIfNotNull(mainSection, "Plugin-Scm-Git-Hash", getGitHeadSha1(false));
```

Comment:
useful for tracking SNAPSHOT deployments.  

Possibly helpful for testing SNAPSHOT builds in the PCT.